ebay-frontend/src/app/upcoming-events-screen/upcoming-events-screen.component.scss

```diff
@@ -1,7 +1,57 @@
 .upcoming-card {
+        background-color: #FFFFFF;
+        display: flex;
+        justify-content: space-between;
+       
+
+}
+.left {
+  background: white;
+}
+.right {
+  background: white;
+  color: gray;
+  line-height: 100px;
+ 
 }
 
 .upcoming-header-image {
   background-image: url(https://avatars1.githubusercontent.com/u/14041091?s=460&v=4);
   background-size: cover;
+
+}
+
+
+.upcoming-card:hover {
+  box-shadow: 0 6px 10px 0 #668;
+  transform: scale(1.0);
+}
+
+
+.createEventButton {
```

Comment:
None of these except the last three should be needed if you're using mat-fab-button correctly